cylc/flow/cfgspec/suite.py

```diff
@@ -1472,6 +1483,23 @@ def upg(cfg, descr):
             ['runtime', '__MANY__', 'job', job_setting],
             ['runtime', '__MANY__', job_setting]
         )
+    for job_setting in [
+        'execution polling intervals',
+        'submission polling intervals',
+        'submission retry delays'
+    ]:
+        u.deprecate(
+            '8.0.0',
+            ['runtime', '__MANY__', job_setting],
+            ['runtime', '__MANY__', job_setting],
+            cvtr=converter(
+                lambda x: x, (
+                    "Config set in global.cylc[platforms].\n"
+                    "Currently this item will over-ride the platform config, "
+                    "but this config will be obsolete at Cylc 9."
+                )
+            )
+        )
```

Comment:
Instead of getting this verbose output:
```
WARNING -  * (8.0.0) [runtime][foo, cat, dog][execution polling intervals] ->
    [runtime][foo, cat, dog][execution polling intervals ...
```
would it be possible to give a warning like how it's done here:
https://github.com/cylc/cylc-flow/blob/078f45c26bb52ef62218488a3297520d075c8af0/cylc/flow/cfgspec/suite.py#L1529-L1530

Reply:
Yes, Happy to have a look and see if I can do that. Sounds like a good idea.

Reply:
I didn't even make it that complex. Just logged a warning.